### ISSUE TITLE
Upgrade codebase to python 3

### ISSUE THREAD

**Issue by Link-** [Link-/pyboiler#10]
Update the entire codebase to python 3

**Comment by HushmKun** [comment]
Hello @Link- ! 

Is this project abandoned ?

**Comment by Link-** [comment]
Yes @HushmKun it is :) that reminds me, I should probably archive it!

**Comment by HushmKun** [comment]
> Yes @HushmKun it is :) that reminds me, I should probably archive it!

Is It okay If I kinda of fork it for the python 3 ?

**Comment by Link-** [comment]
Go for it @HushmKun
